### PR TITLE
Add support for other languages

### PR DESCRIPTION
**Pull request by lucasoshiro**
Hi!

Although the https://github.com/tldr-pages/tldr repository has pages in other languages than English, the only way to read a page in another language was changing the page source with `-s` or `--source`.

This PR makes `tldr` detect the system language and use it to download the page in that language if it's available. Otherwise, it will download in English instead. It's also possible to provide the language with `-l` or `--language`.

### CODE REVIEW

**Comment by lucasoshiro** [comment]
ping @waldyrious ! Can you review this, please? Thanks!

**Comment by ivanhercaz** [comment]
This PR is very interesting. When I have a bit of time I could review it, but I can't promise it, @lucasoshiro.

**Comment by lucasoshiro** [comment]
@ivanhercaz ok, thanks!

**Comment by zdroid** [comment]
@lucasoshiro I wasn't active much when you proposed this, so it seems I missed this PR.

This has some merge conflicts with the recently merged PRs that changed the way page requests operate. Are you still interested in working on this, to check if those recent changes work along with your proposed change?

Thank you for the valuable contribution nevertheless.

**Comment by lucasoshiro** [comment]
@zdroid I rebased this PR with master and solved the conflitcts, I think it's ok now. Thanks for the review!

**Comment by zdroid** [comment]
What do you think about adding another TLDR_ configuration variable? Maybe `TLDR_PAGES_LANGUAGE` or just `TLDR_LANGUAGE`.

**Comment by MasterOdin** [comment]
The client specification already includes words on how to support language environment variables: https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#language.

File: tldr.py [hunk 1]
```diff
@@ -137,20 +139,35 @@ def have_recent_cache(command, platform):
         return False
 
 
-def get_page_url(platform, command, remote=None):
+def get_page_url(platform, command, remote=None, language=None):
     if remote is None:
         remote = PAGES_SOURCE_LOCATION
-    return remote + "/" + platform + "/" + quote(command) + ".md"
+
+    if language is None:
```
Comment (MasterOdin) [inline tldr.py:146]:
```suggestion
    if language is None or language == 'en':
        return None
    else:
        language = '.' + language
```

**Comment by zdroid** [comment]
@MasterOdin I decided to merge #101 first because else we'd be making too many redundant changes. But we can move on to complete this one now.

File: tldr.py [hunk 2]
```diff
@@ -75,20 +77,35 @@ def have_recent_cache(command, platform):
...
+        language = ''
```
Comment (MasterOdin) [inline tldr.py:85]:
```
    if language is None or language == 'en':
        language = ''
```

File: tldr.py [hunk 3]
```diff
@@ -26,6 +27,7 @@
     'TLDR_DOWNLOAD_CACHE_LOCATION',
     'https://tldr-pages.github.io/assets/tldr.zip'
 )
+DEFAULT_LANG = locale.getlocale()[0]
```
Comment (MasterOdin) [inline tldr.py:30]:
```suggestion
DEFAULT_LANG = os.environ.get('LANG', locale.getlocale()[0])
```

**Pull request merged**